config/routes.rb

```diff
@@ -296,7 +296,11 @@
         :via => :get
   ####
 
+  #### PublicBodyChangeRequest controller
   resource :change_request, :only => [:new, :create], :controller => 'public_body_change_requests'
+  match 'change_request/new/:body' => 'public_body_change_requests#new',
+        :as => :new_change_request_body,
+        :via => :get
```

Comment:
I need to double check but I think that would prevent  `/change_request/new` without the public body information (currently the default) for requesting that a new body is added